<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EdadForm</class>
 <widget class="QGroupBox" name="EdadForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>60</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Edad</string>
  </property>
  <property name="title">
   <string>Edad</string>
  </property>
  <widget class="QLineEdit" name="cuentaEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="edadAnyoLbl">
      <property name="text">
       <string>AÃ±o:</string>
      </property>
      <property name="buddy">
       <cstring>edadAnyoCmb</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="edadAnyoCmb">
      <item>
       <property name="text">
        <string>Otro</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edadAnyoEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="maxLength">
       <number>5</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>edadAnyoCmb</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>EdadForm</receiver>
   <slot>cambio(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cambio(int)</slot>
 </slots>
</ui>
